pkg/services/rpcsrv/client_test.go

```diff
@@ -2466,6 +2466,26 @@ func TestClient_GetVersion_Hardforks(t *testing.T) {
 	require.InDeltaMapValues(t, expected, v.Protocol.Hardforks, 0)
 }
 
+func TestClient_GetVersion_AllFields(t *testing.T) {
+	chain, orc, cfg, logger := getUnitTestChainWithCustomConfig(t, false, false, func(cfg *config.Config) {
+		cfg.ApplicationConfiguration.KeepOnlyLatestState = true
+		cfg.ApplicationConfiguration.SaveInvocations = true
+	})
+
+	_, _, httpSrv := wrapUnitTestChain(t, chain, orc, cfg, logger)
+
+	client, err := rpcclient.New(context.Background(), httpSrv.URL, rpcclient.Options{})
+	require.NoError(t, err)
+	t.Cleanup(client.Close)
+	require.NoError(t, client.Init())
+
+	version, err := client.GetVersion()
+	require.NoError(t, err)
+
+	require.Equal(t, version.Application.SaveInvocations, true)
+	require.Equal(t, version.Application.KeepOnlyLatestState, true)
```

Comment:
Swap the arguments, the first argument is *expected*, the second argument is *actual*.

Reply:
And BTW, just use `require.True` instead.